google/cloud/documentai_toolbox/wrappers/hocr.py

```diff
@@ -0,0 +1,221 @@
+    return document_text[start_index:end_index].replace('/n', '')
+
+
+def _load_hocr_words(words: List[documentai.Document.Page.Token], document_text: str, dimensions: documentai.Document.Page.Dimension):
+    w = []
+    for word in words:
+        text = _get_text(word, document_text)
+        w.append(_HocrWord(text=text, documentai_word=word, dimensions=dimensions))
+    else:
+        return w
+
+
+def _load_hocr_lines(lines: List[documentai.Document.Page.Line], page: documentai.Document.Page, document_text: str):
+    l = []
+    for line in lines:
+        start_index = line.layout.text_anchor.text_segments[0].start_index
+        end_index = line.layout.text_anchor.text_segments[0].end_index
+        words = [word for word in page.tokens if word.layout.text_anchor.text_segments[0].start_index >= start_index if word.layout.text_anchor.text_segments[0].end_index <= end_index]
+        l.append(_HocrLine(words=words, text=(_get_text(line, document_text)), documentai_line=line, page=page, dimensions=(page.dimension), document_text=document_text))
+    else:
+        return l
+
+
+def _load_hocr_paragraphs(paragraphs: List[documentai.Document.Page.Paragraph], page: documentai.Document.Page, document_text: str):
+    p = []
+    for paragraph in paragraphs:
+        start_index = paragraph.layout.text_anchor.text_segments[0].start_index
+        end_index = paragraph.layout.text_anchor.text_segments[0].end_index
+        lines = [line for line in page.lines if line.layout.text_anchor.text_segments[0].start_index >= start_index if line.layout.text_anchor.text_segments[0].end_index <= end_index]
+        p.append(_HocrParagraph(lines=lines, page=page, dimensions=(page.dimension), document_text=document_text, documentai_paragraph=paragraph))
+    else:
+        return p
+
+
+def _load_hocr_blocks(page: documentai.Document.Page, document_text: str):
+    blocks = []
+    for block in page.blocks:
+        start_index = block.layout.text_anchor.text_segments[0].start_index
+        end_index = block.layout.text_anchor.text_segments[0].end_index
+        paragraphs = [p for p in page.paragraphs if p.layout.text_anchor.text_segments[0].start_index >= start_index if p.layout.text_anchor.text_segments[0].end_index <= end_index]
+        blocks.append(_HocrBlock(paragraphs=paragraphs, page=page, dimensions=(page.dimension), document_text=document_text, documentai_block=block))
+    else:
+        return blocks
+
+
+def _load_pages(pages: List[documentai.Document.Page], document_text: str, filename: str):
+    res_pages = []
+    for page in pages:
+        res_pages.append(_HocrPage(documentai_page=page.documentai_page, document_text=document_text, filename=filename))
+
+    return res_pages
+
+
+@dataclasses.dataclass
+class _HocrWord:
+    text: str = dataclasses.field(repr=False)
+    documentai_word: documentai.Document.Page.Token = dataclasses.field(repr=False)
+    dimensions: documentai.Document.Page.Dimension = dataclasses.field(repr=False)
```

Comment:
This might be an `InitVar` following the pattern introduced in https://github.com/googleapis/python-documentai-toolbox/pull/110

(The word does not need this as an attribute, and it is used only in `__post_init__` to calculate the bounding box.)

Reply:
yea once the #110 changes are merged i'll modify this PR to use the new pattern